Dockerfile

```diff
@@ -5,56 +5,48 @@ ENV LANG en_US.utf8
 
 RUN yum install -y epel-release \
         && yum install -y \
-                which \
-                make \
-                gcc \
-                clang \
-                scons \
-                swig \
-                python36-devel \
-                python36 \
-                nss-devel \
-                msgpack-devel \
-                jq \
-                java-1.8.0-openjdk \
-                parallel \
-                tree \
+        which \
+        make \
+        gcc \
+        clang \
+        scons \
+        swig \
+        python36-devel \
+        python36 \
+        nss-devel \
+        msgpack-devel \
+        jq \
+        java-1.8.0-openjdk \
+        parallel \
+        tree \
         && yum clean all \
         && rm -rf /var/cache/yum
 
 # symbolically link to name without version suffix for libprio
 RUN ln -s /usr/include/nspr4 /usr/include/nspr \
-    && ln -s /usr/include/nss3 /usr/include/nss
+        && ln -s /usr/include/nss3 /usr/include/nss
+
+COPY ./google-cloud-sdk.repo /etc/yum.repos.d/
+RUN yum install -y google-cloud-sdk
+RUN gcloud config set disable_usage_reporting true
 
 # prepare the environment for testing in development
 ENV PATH="$PATH:~/.local/bin"
-RUN python3 -m ensurepip && pip3 install tox setuptools wheel black
-
-RUN curl https://sdk.cloud.google.com | bash
-ENV PATH="$PATH:~/google-cloud-sdk/bin"
-RUN gcloud config set disable_usage_reporting true
+RUN python3 -m ensurepip && pip3 install setuptools wheel black
 
 # install the app
 WORKDIR /app
 ADD . /app
 
-# Build the prio wrapper
-WORKDIR /app/prio
-RUN make
-
-WORKDIR /app/processor
-RUN python3 setup.py bdist_egg
-
-WORKDIR /app
+# NOTE: force reinstall is temporary until the package is published on pypi
 RUN pip3 install --upgrade pip && \
         pip3 install -r requirements.txt
 
 ENV SPARK_HOME=/usr/local/lib/python3.6/site-packages/pyspark
 ENV PYSPARK_PYTHON=python3
 
-CMD cd prio && tox && cd .. && \
-        cd processor && tox && cd .. \
-        prio/scripts/test-cli-integration && \
+CMD pytest -v tests && \
+        scripts/test-cli-integration && \
```

Comment:
Sorry if I've asked this before, but isn't it a little strange to run tests while building the docker container?

Reply:
This is `CMD` which sets the default command to be run on startup, which you might be confusing with `RUN`.